src/gluonts/model/seq2seq/_forking_estimator.py

```diff
@@ -154,7 +155,7 @@ def __init__(
         enable_encoder_dynamic_feature: bool = True,
         enable_decoder_dynamic_feature: bool = True,
         trainer: Trainer = Trainer(),
-        scaling: bool = False,
+        scaling: Optional[bool] = None,
         scaling_decoder_dynamic_feature: bool = False,
```

Comment:
Do we want to turn on scaling of the dynamic features or this should be unrelated to the distribution change right?

Reply:
I think this is unrelated maybe, we could do it but maybe as a separate story